### PR TITLE
Add OpsGenie v2 to UI

### LINKED ISSUE #2929: Update Alert Node Config for OpsGenie V2
connect https://github.com/influxdata/kapacitor/issues/1708

- [x] the stuff from https://github.com/influxdata/chronograf/issues/2786
- [x] Add an OpsGenie v2 config option for the Kapacitor config page
- [x] Add a warning in the UI indicating that OpsGenie v1 is deprecated
- [x] Clicking enter after entering the API key should immediately change to the redacted input
- [x] Update Kapacitor to new rc tag

### PR DESCRIPTION
**Pull request by ischolten**
Closes #2929

_Briefly describe your proposed changes:_
Add config in UI for OpsGenie v2
Update kapacitor dependency to 1.5 rc 2

  - [x] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergeable
  - [x] Tests pass
  - [ ] swagger.json updated (if modified Go structs or API)
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/kapacitor/components/AlertTabs.tsx [hunk 1]
```diff
@@ -47,128 +140,44 @@ class AlertTabs extends Component {
...
-      })
-    )
-  }
-
-  render() {
+  public render() {
     const {configSections} = this.state
     const {hash} = this.props
 
+    // console.log(configSections)
```
Comment (bthesorceror) [inline ui/src/kapacitor/components/AlertTabs.tsx]: Would you mind removing this? 😄

File: ui/src/kapacitor/components/AlertTabs.tsx [hunk 2]
```diff
@@ -345,7 +376,7 @@ class AlertTabs extends Component {
           <TabList customClass="config-endpoint--tabs">
             {_.reduce(
               configSections,
-              (acc, _cur, k) => {
+              (acc, __, k) => {
```
Comment (bthesorceror) [inline ui/src/kapacitor/components/AlertTabs.tsx]: nice!

**Comment by jaredscheib** [comment]
I'm going to merge and let @ischolten update swagger for the new OpsGenie2 config upon her return: https://github.com/influxdata/chronograf/issues/3382.

**Comment by ischolten** [comment]
Swagger was updated by PR #3398

**Pull request merged**